src/strip-loader.ts

```diff
@@ -1,36 +1,42 @@
 import type { LoadFnOutput, LoadHookContext } from "node:module";
-import type { Options } from "../lib/wasm";
+import { isSwcError, wrapAndReThrowSwcError } from "./errors.js";
```

Comment:
An alternative to this might be depending on `./index.js` solely so that we don't need an extra entry point. As `index.js` is a bundled entrypoint, I think this can be left as a follow up.

Reply:
> An alternative to this might be depending on `./index.js` solely so that we don't need an extra entry point. As `index.js` is a bundled entrypoint, I think this can be left as a follow up.

Mh in core I would want to wrap the object is either a syntax error or a Node.js error. So it would be another layer of wrapping